source/type/pb_converter.h

```diff
@@ -380,8 +380,9 @@ struct PbCreator {
         void fill_messages(const nt::HasMessages* nav_obj, P* pb_obj){
             if (nav_obj == nullptr) {return ;}
             if (dump_message_options.dump_message == DumpMessage::No) { return; }
-            for (const auto& message : nav_obj->get_applicable_messages(pb_creator.now,
-                                                                        pb_creator.action_period)){
+            const bool dump_line_sections = dump_message_options.dump_line_section == DumpLineSectionMessage::Yes;
```

Comment:
Minor (readability): add parenthesis

Reply:
I don't think it helps the understanding